### ISSUE TITLE
Hamburger menu has no visible focus when focussed with keyboard

### ISSUE THREAD

**Issue by EmmaDawsonDev** [EddieHubCommunity/LinkFree#4226]
### Description

When tabbing through the site it should be possible to see which element is in focus by the presence of a visible focus outline. The hamburger menu and its corresponding close button are missing this outline.

### Screenshots

<img width="329" alt="Screenshot 2023-01-29 165736" src="https://user-images.githubusercontent.com/57045550/215338624-ebd2a145-cdcc-4ddc-baa4-d2a777d60a2f.png">

**Comment by sidharthviz** [comment]
Can i work on it ?

**Comment by EmmaDawsonDev** [comment]
Sure @sidharthviz

**Comment by alokVerma749** [comment]
PR: #4245  solves this issue

**Comment by sidharthviz** [comment]
> PR: #4245 solves this issue

Hi @alokVerma749 Please work on those issues which you've been assigned for, Thank you

**Comment by alokVerma749** [comment]
Hi @sidharthviz I apologize for not communicating my plans beforehand. I had noticed that this issue had been open for a while and I wanted to take action to move it forward. In the future, I will make sure to check before taking on any issues.

**Comment by sidharthviz** [comment]
> 

@alokVerma749  No worries! Good for you 👍🏼

**Comment by EmmaDawsonDev** [comment]
@alokVerma749 please don’t make PRs for issues already assigned to other people, it’s bad etiquette. This issue was only created yesterday so it hadn’t been open “for a while” like you state. Please give the assignee time to work on things. For many this is their first contribution to open source and it can be discouraging to have this happen.

**Comment by alokVerma749** [comment]
My apologies for this conduct. As I am new to contributing
